content/numpy/concepts/math-methods/terms/cos/cos.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: 'NumPy cos'
+Description: 'Returns the cos of the argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'NumPy'
+  - 'Trig'
+  - 'Math'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+In NumPy, the **cos** function computes the cosine of each element in an array. This trigonometric function is essential for various mathematical computations, especially in physics, engineering, computer graphics, and signal processing, and more.
```

Comment:
```suggestion
In NumPy, the **`.cos()`** function computes the cosine of each element in an array. This trigonometric function is essential for various mathematical computations, especially in physics, engineering, computer graphics, signal processing, and more.
```

Reply:
In a PR, whenever you introduce the term that you have written a PR for, make sure it is in **bold**, and if its a term like this one make sure it is in backticks(` `) and bold, like: **`.cos()`**